src/Symfony/Component/Notifier/Bridge/OvhCloud/OvhCloudTransport.php

```diff
@@ -122,7 +125,7 @@ protected function doSend(MessageInterface $message): SentMessage
         if (200 !== $statusCode) {
             $error = $response->toArray(false);
 
-            throw new TransportException(\sprintf('Unable to send the SMS: %s.', $error['message']), $response);
+            throw new TransportException(\sprintf('Unable to send the SMS: "%s".', $error['message'] ?? 'Unknown error'), $response);
```

Comment:
CS-fixer false-positive:
```suggestion
            throw new TransportException(\sprintf('Unable to send the SMS: %s.', $error['message'] ?? 'Unknown error'), $response);
```

Reply:
The ci won't pass with the old version even if I don't update the line: https://fabbot.io/report/symfony/symfony/59481/58c0e8537fc8f2c55f4e7292f771faf9fd1d4d82